packages/template-retail-react-app/app/hooks/use-add-to-cart-modal.js

```diff
@@ -105,77 +113,162 @@ export const AddToCartModal = () => {
+                                            borderBottomWidth={{base: '1px', lg: '0px'}}
+                                            borderColor="gray.200"
+                                            borderStyle="solid"
+                                            data-testid="product-added"
+                                        >
+                                            <Flex gridGap="4">
+                                                <Box w="24" flex="none">
+                                                    <AspectRatio ratio="1">
+                                                        <img src={image.link} alt={image.alt} />
+                                                    </AspectRatio>
+                                                </Box>
+
+                                                <Box>
+                                                    <Text fontWeight="700">{product.name}</Text>
+                                                    <Box
+                                                        color="gray.600"
+                                                        fontSize="sm"
+                                                        fontWeight="400"
+                                                    >
+                                                        {Object.entries(
+                                                            variationAttributeValues
+                                                        ).map(([name, value]) => {
                                                             return (
                                                                 <Text key={value}>
                                                                     {name}: {value}
                                                                 </Text>
                                                             )
-                                                        }
-                                                    )}
-                                                    <Text>
-                                                        {intl.formatMessage({
-                                                            defaultMessage: 'Qty',
-                                                            id: 'add_to_cart_modal.label.quantity'
                                                         })}
-                                                        : {quantity}
-                                                    </Text>
+                                                        <Text>
+                                                            {intl.formatMessage({
+                                                                defaultMessage: 'Qty',
+                                                                id: 'add_to_cart_modal.label.quantity'
+                                                            })}
+                                                            : {quantity}
```

Comment:
Notably pluralization in LTR (e.g. Japanese, which we have as a locale) would break here, but this is not a critique of your PR, just something we're not handling for and could be mindful of fixing in the future